include/openssl/crypto.h

```diff
@@ -245,6 +245,54 @@ typedef struct {
     struct CRYPTO_dynlock_value *data;
 } CRYPTO_dynlock;
 
+#if !defined(OPENSSL_THREADS) || defined(CRYPTO_TDEBUG)
+typedef int CRYPTO_MUTEX;
+typedef int CRYPTO_ONCE;
+typedef int CRYPTO_THREAD_LOCAL;
+typedef int CRYPTO_THREAD_ID;
+
+#  define CRYPTO_ONCE_STATIC_INIT 0
```

Comment:
Perhaps I'm conflating the openssl style guide with one from a different project, but I would expect to see:

    #if <condition>
    # define blah
    #elif <condition2>
    # if <condition3>
    #  define foo
    # endif
    #endif